locale/de/Factorissimo.cfg

```diff
@@ -6,4 +6,9 @@ factory-power-receiver=Fabrikstromempfänger
 factory-power-distributor=Fabrikstromverteiler
 
 [technology-name]
-factory-architecture=Architektur
\ No newline at end of file
+factory-architecture=Architektur
+
+[tile-name]
+factory-wall=Fabrikmauer
+factory-floor=Fabrikhalle
```

Comment:
Better translation for factory floor, if referring to the floor of the factory would be fabrikboden  as fabrikhalle would refer to the entire factory building, and or more precisely to one building /hall